### PR TITLE
Small batch of updates to the SPDX landscape 🤞🏻

### PR DESCRIPTION
**Pull request by MordodeMaru**
A small batch of updates to the landscape now that I THINK I know how the landscape contribution process works. I have checked all the boxes below because I don't want to upset the YAML gods anymore 🙏🏻

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [X] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [X] Is your project closed source or, if it is open source, does your project have at least 250 GitHub stars?
* [X] Have you picked the single best (existing) category for your project?
* [X] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [X] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
* [X] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [X] Does your project/product name match the text on the logo?
* [X] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [X] ~10 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by MordodeMaru** [comment]
I pray to @tracymiranda this thing works.

**Comment by jorydotcom** [comment]
@jmertic can you possibly help troubleshoot this PR if you have a moment?

**Comment by jmertic** [comment]
Looks like these are the errors:

```
6:23:02 AM: FATAL: No cached entry, and Kubernetes has issues with logo: Kubernetes_logo.png, ENOENT: no such file or directory, open '/opt/repo/hosted_logos/Kubernetes_logo.png'
6:23:02 AM: FATAL: Tern either has no crunchbase entry or it is invalid
6:23:02 AM: FATAL: bom either has no crunchbase entry or it is invalid
6:23:02 AM: FATAL: Augur either has no crunchbase entry or it is invalid
```

For the first, just make sure the logo is uploaded and named correctly.

For the others - you need to add a `crunchbase:` entry or if one doesn't exist add this:

```
organization:
    name: NAME
```

Hopefully that helps!

File: landscape.yml [hunk 1]
```diff
@@ -74,4 +94,13 @@ landscape:
             homepage_url: https://github.com/tern-tools/tern
             repo_url: https://github.com/tern-tools/tern
             logo: tern.svg
-            crunchbase: https://www.crunchbase.com/organization/vmware
+          - item:
+            name: bom
+            homepage_url: https://github.com/kubernetes-sigs/bom
+            repo_url: https://github.com/kubernetes-sigs/bom
+            logo: bom.svg
```
Comment (jmertic) [inline landscape.yml:101]: Add line after this:
```
crunchbase: https://www.crunchbase.com/organization/cloud-native-computing-foundation
```

File: landscape.yml [hunk 2]
```diff
@@ -74,4 +94,13 @@ landscape:
...
+          - item:
+            name: Augur
+            homepage_url: http://www.augurlabs.io
+            repo_url: https://github.com/chaoss/augur
+            logo: Augurlabs.svg
```
Comment (jmertic) [inline landscape.yml:106]: Add line after this:

```
crunchbase: https://www.crunchbase.com/organization/linux-foundation
```

File: landscape.yml [hunk 3]
```diff
@@ -34,7 +34,27 @@ landscape:
...
+            crunchbase: https://www.crunchbase.com/organization/vmware-tanzu
+          - item:
+            name: GitHub
+            homepage_url: https://github.com/features
+            logo: GitHub_Logo.svg
+            crunchbase: https://www.crunchbase.com/organization/github
+          - item:
+            name: Kubernetes
+            homepage_url: https://kubernetes.io/
+            logo: Kubernetes_logo.png
```
Comment (jmertic) [inline landscape.yml:56]: Change to:

```
logo: Kubernetes_logo.svg
```

File: landscape.yml [hunk 4]
```diff
@@ -74,4 +94,13 @@ landscape:
             homepage_url: https://github.com/tern-tools/tern
             repo_url: https://github.com/tern-tools/tern
             logo: tern.svg
-            crunchbase: https://www.crunchbase.com/organization/vmware
```
[the remark this answers is not in the document]
Comment (MordodeMaru) [reply landscape.yml:77]: No because that's the Tern entry, not VMware's. VMware's entry has the Crunchbase line properly added.

**Comment by MordodeMaru** [comment]
Thanks!
> 
> For the others - you need to add a `crunchbase:` entry or if one doesn't exist add this:
> 
> ```
> organization:
>     name: NAME
> ```

This ☝🏻 I don't get. For those entries that don't have a Crunchbase entry I need to add a line called name:NAME?? I'm not sure this is right.

**Comment by MordodeMaru** [comment]
> For the first, just make sure the logo is uploaded and named correctly.

🤦🏽‍♂️ Done. Thanks!

File: landscape.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (jmertic) [reply landscape.yml:77]: You need a `crunchbase:` entry for every item; you should put in the crunchbase for the organization owning or the primary maintainer for the project.

**Comment by jmertic** [comment]
> Thanks!
> 
> > For the others - you need to add a `crunchbase:` entry or if one doesn't exist add this:
> > ```
> > organization:
> >     name: NAME
> > ```
> 
> This ☝🏻 I don't get. For those entries that don't have a Crunchbase entry I need to add a line called name:NAME?? I'm not sure this is right. Below is what Andrey Kozlov said in the CNCF's landscapers.
> 
> CleanShot 2023-08-25 at 10 13 19

In general - you need either `crunchbase:` or `organization:` and then `name:`. 

However, each of those project's owner/primary maintainer has a crunchbase profile, so just use the ones I suggested.

**Comment by tracymiranda** [comment]
LGTM! Thanks for persisting with this @MordodeMaru - got there in the end!

**Pull request merged**